modules/UI/videolayout/VideoContainer.js

```diff
@@ -576,6 +573,17 @@ export class VideoContainer extends LargeContainer {
     }
 
     /**
+     * Sets the blur background to be invisible and pauses any playing video.
+     *
+     * @private
+     * @returns {void}
+     */
+    _hideVideoBackground() {
+        this.$videoBackground.css({ visibility: 'hidden' });
```

Comment:
That's great that you have changed `.hide()` to just a style! A long time ago I tested some performance and `.hide()` was ~50 time slower than `css({display: none})` :)